src/index.js

```diff
@@ -206,17 +216,18 @@ const generateCriticalCssWrapped = async function generateCriticalCssWrapped (
         width,
         height,
         forceInclude,
+        propertiesToRemove: propertiesToRemove,
```

Comment:
remove the `: propertiesToRemove` - not needed with modernJS.

Reply:
also just nitpick: given the `// postformatting`  comment further down, and that this properties relate to postformatting - please move it to be below that comment.

Reply:
you are absolutly right